### ISSUE TITLE
[RFE] shellsyntax: Ignore Tcl scripts that pretend to be shell scripts

### ISSUE THREAD

**Issue by msrb** [rpminspect/rpminspect#1411]
**Is your feature request related to a problem? Please describe.**

For some reason, Tcl scripts often pretend to be shell scripts, but they are not. This seems to be a common practice in Tcl scripts:

```
#!/bin/sh
# Tcl ignores the next line -*- tcl -*- \
exec wish "$0" -- "$@"

package require Tk

proc hasworktree {} {
    return [expr {[exec git rev-parse --is-bare-repository] == "false" &&
                  [exec git rev-parse --is-inside-git-dir] == "false"}]
}

```
This is from git package, specifically from `/usr/bin/gitk`.

Some discussion about this in https://stackoverflow.com/questions/18781355/what-is-the-need-for-shell-when-executing-tclsh

**Describe the solution you'd like**

If possible, the shellsyntax inspection would be smart enough to ignore Tcl scripts and don't report syntax problems in them.

**Describe alternatives you've considered**

Ignoring Tcl scripts via local rpminspect.yaml is possible.

**Additional context**

Reported by a RHEL package maintainer.
